doc/analysis-plan.md

```diff
@@ -0,0 +1,54 @@
+# Possible analyses
+### I. How to answer research questions 
+For variable community composition, we plan to plot them with PCA and spatial autocorrelation, identify dominant genus (simpson's index), abundance/importance (if have over 10 genus).
+
+	Dependent variables: community composition for each site, species richness, species evenness. 
+	Independent variables includes: pH, seasonality, sites (other environmental factor might be added). 
+
+We predict that the independent variables will affect the dependent variables, and conservation action will be dependent on the community composition, species richness and speices evenness of the sites. 
+
+Use mixed models (or structure equation model) for the realationship between dependent variables and independent variables 
+
+PCA of abundance - community composition for each site
+
+MANOVA - community composition similarity across sites 
+
+Spatial autocorrelation for sites with dependent variabels (pH, seasonality)
+
+
+#### II. How do we formulate the data
+
+ **Cleaning/transformation plan:**
+
+Merge data from each year and season (all years, all sites, might take an average across the years)
```

Comment:
I am not sure if we should take an average across the years at this point before we analyze them.